spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitManagementTemplate.java

```diff
@@ -70,6 +70,20 @@ public RabbitManagementTemplate(Client rabbitClient) {
 
 	/**
 	 * Construct a template using the supplied uri.
+	 * @param uri the uri - must include user info, e.g.
+	 * "http://guest:guest@localhost:15672/api/".
```

Comment:
Yes, we can use the new ctor - however, I'd like to cherry-pick this to 1.6.x  - we need to discuss with Michael when 1.0.1.RELEASE will be available.